projects/cashmere/src/lib/sass/button.scss

```diff
@@ -67,6 +62,30 @@ $btn-icon-sz: 18px;
     }
 }
 
+@mixin hc-button-sm() {
+    padding: 3px 12px;
+    height: 26px;
+    min-width: auto;
+    font-size: 14px;
+    line-height: 14px;
+}
+
+@mixin hc-button-md() {
+    padding: 0 20px;
+    height: 35px;
+    min-width: 150px;
+    font-size: 15px;
+    line-height: 15px;
+}
+
+@mixin hc-button-lg() {
+    padding: 0 30px;
+    height: 45px;
```

Comment:
For `lg` I'd suggest we do 40px rather than 45.  45 is pretty huge and I'm sure sure I'd ever see a use for that.  But in prototypes before I have used 40px buttons, like on the close button at the bottom of a modal.